### PR TITLE
Bulk export

### PR DESCRIPTION
**Pull request by askeron**
This adds buttons to bulk export all chats at once. (see issue #36 )
Disclamer: This is not my code, I found this commit in the fork of zmeyc. I just also need this functionality.

### CODE REVIEW

**Comment by andreas-mausch** [comment]
Wow thanks. Have you tested it and verified its functional?

I really need to set up my dev environment again..

One thing I've noticed on the first look: it seems to replace the old buttons, right? I'd prefer if it would add new buttons instead

**Comment by askeron** [comment]
Sadly, no. I'm a Developer, but I do mostly Java, Kotlin and NodeJS. The last time I wrote C and C++ was in collage. I tried to compile your code with Visual Studio 2019, but failed because there seems to be several thing that still link to Visual Studio 2017 and 2010. (At least the error messages indicated this.) And I got this error with your code independently from the new code, so that I assume it is only because of my building environment.
But I looked through the diffs and I am understanding most of the changes. They seem legit and as I understand the diff, he did not remove the single export buttons but added additional buttons for bulk export. He only refactored the buttons so that both buttons could use the same subfuction.

**Review by david824 — APPROVED** [review]

**Review by david824 — APPROVED** [review]

**Comment by andreas-mausch** [comment]
Hi @askeron,

I've merged the changes on command-line and resolved the conflicts with the newest master.
However, github now doesn't recognize the changes are already in master, and your name doesn't appear in the git log.
I'm sorry about that. :(
(If you know how to fix it, please let me know)

Everything worked without any changes, so thank you very much for the contribution.

**Comment by askeron** [comment]
That's fine for me. The important things is that it made it in the master.

File: resources/Resources.rc [hunk 1]
```diff
@@ -199,6 +199,10 @@ BEGIN
         MENUITEM "Decrypt .crypt&8...\tCtrl+8", ID_MENU_MAIN_FILE_DECRYPT_CRYPT8
         MENUITEM "Decrypt .crypt&12...",        ID_MENU_MAIN_FILE_DECRYPT_CRYPT12
         MENUITEM SEPARATOR
+        MENUITEM "Export All Chats To Txt...",     ID_MENU_MAIN_FILE_EXPORT_ALL_CHATS_TXT
```
Comment (cwhite72-droid) [inline resources/Resources.rc:202]: Run

**Pull request closed without merging**